src/mapml.css

```diff
@@ -113,6 +113,7 @@
 
 .leaflet-control-layers-list {
   padding: 0;
+  text-transform: capitalize;
```

Comment:
@prushforth this was inspired by https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/164/commits/a99b26db61d833222278307fe410e5aa5f26639f#diff-17554ee353b2637e383832c72c310b60R728, let me know if you don't agree that text in the layer control should be capitalized by default. I think it's nice for a consistent look, but perhaps there are cases where this isn't desired?